src/components/Header/index.tsx

```diff
@@ -0,0 +1,252 @@
+import Button from '../Button';
+import Container from '../Container';
+import Fixed from '../Fixed';
+import { Flex } from '../Flex';
+import Img from '../Img';
+import Link from '../Link';
+import Txt from '../Txt';
+import { NavDropDown } from './NavDropDown';
+
+const NavigationItem = styled(Box)`
+	margin: 0;
+
+	a {
+		margin: 0 !important;
+		color: ${(props) => props.theme.colors.text.main};
+		display: block !important;
+		padding-top: 12px !important;
+		padding-bottom: 10px !important;
+		font-size: initial;
+
+		&:hover:before {
+			border: none;
+		}
+	}
+
+	svg {
+		float: right;
+		transform: translateY(4px);
+
+		path {
+			fill: ${(props) => props.theme.colors.text.main};
+		}
+	}
+`;
+interface HeaderProps extends BoxProps {
+	logo?: string;
+	logoAltText?: string;
+	rootRoute?: string;
+	routes: Route[];
+	actions?: React.ReactElement;
+}
+
+interface Route {
+	title: string;
+	showDividers?: boolean;
+	path?: string;
+	blank?: boolean;
+	routes: Route[];
+}
+const getNavigationRoutes = (routes: Route[]) => {
+	return routes.map((route, index) => {
+		if (route['routes']) {
+			return (
+				<NavigationItem pl={[0, 0, 3, 3]} key={index} mx={15}>
+					<NavDropDown title={route.title} showDividers={route.showDividers}>
+						{getNavigationRoutes(route.routes)}
+					</NavDropDown>
+				</NavigationItem>
+			);
+		}
+
+		return (
+			<NavigationItem pl={[0, 0, 3, 3]} key={index} mx={[2, 2, 15]}>
+				<Link
+					blank={route.blank || false}
+					href={route.path}
+					style={{
+						padding: '3px 0 3px',
+						display: 'auto',
+					}}
+					color="text.main"
+					fontSize={13}
+				>
+					<Txt.span bold>{route.title}</Txt.span>
+				</Link>
+			</NavigationItem>
+		);
+	});
+};
+
+export const Wrapper = styled.div`
+	position: relative;
+	width: 100%;
+	background: white;
+	z-index: 5;
+
+	+ div:first-of-type {
+		margin-top: 80px;
+	}
+
+	@media (max-width: ${(props) =>
+			invertBreakpoint(props.theme.breakpoints[1])}px) {
+		position: fixed;
+		background: white;
+		box-shadow: 0px 0px 21px 5px rgba(152, 173, 227, 0.2);
+	}
+`;
+
+const Actions = styled(Box)`
+	justify-content: flex-end;
+	align-items: center;
+`;
+
+const MobileActions = styled(Flex)`
+	display: none;
+	min-height: 84px;
+
+	@media (max-width: ${(props) =>
+			invertBreakpoint(props.theme.breakpoints[1])}px) {
+		display: flex;
+	}
+`;
+
+const NavigationItems = styled(Flex)<{ menuOpen?: boolean }>`
+	@media (max-width: ${(props) =>
+			invertBreakpoint(props.theme.breakpoints[1])}px) {
+		position: fixed;
+		background: white;
+		top: 0;
+		right: 0;
+		width: 300px;
+		bottom: 0;
+		justify-content: flex-start;
+		transform: translateX(100%);
+		transition: transform 200ms ease-in-out;
+		padding: 16px;
+		overflow: auto;
+		flex-wrap: nowrap;
+
+		${(props) =>
+			props.menuOpen
+				? css`
+						position: fixed;
+						transform: translateX(0%);
+				  `
+				: ''}
+	}
+`;
+
+export const Navbar = styled(Flex)`
+	min-height: 80px;
```

Comment:
All of the styling for Navbar seems like it should be done using the styled-system props instead